.github/workflows/deploy-ui-components.yml

```diff
@@ -40,32 +28,19 @@ jobs:
     runs-on: ubuntu-latest
     environment: ${{ inputs.environment }}
     name: ${{ inputs.name }}
-    env:
-      VITE_EVERVAULT_JS_URL: ${{ inputs.vite-evervault-js-url }}
-      VITE_KEYS_URL: ${{ inputs.vite-keys-url }}
-      VITE_API_URL: ${{ inputs.vite-api-url }}
```

Comment:
These need to be set when building ui components so we will need to be sure to add these to the environment vars in action settings.